test/honeybadger_test.exs

```diff
@@ -159,6 +159,17 @@ defmodule HoneybadgerTest do
       assert "CustomError" = error["class"]
       assert "a message" = error["message"]
     end
+
+    test "sending a notice when the message is an improper list of iodata" do
+      restart_with_config(exclude_envs: [])
+
+      message = ["RealError", 32, 40, "#PID<0.1.0>" | " ** Error"]
+
+      Honeybadger.notify(%RuntimeError{message: message})
+
+      assert_receive {:api_request, %{"error" => error}}
+      assert error["message"] == "RealError (#PID<0.1.0> ** Error"
```

Comment:
Out of curiosity, why the unbalanced parentheses here? CI seems to pass so I assume it’s not a typo.

Reply:
The open parens came from two integers in the iodata. The last value in the improper list is a binary with a made up error. Together they show that iodata is getting converted correctly, even though the message is totally fake.

Reply:
Ah, doh. Cheers :)